docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
+The Hibernate ORM with Panache extension contains the following classes:
 
-[source,xml]
+* `PanacheEntity`
+* `PanacheEntityBase`
+* `PanacheQuery`
+* `PanacheRepository`
+* `PanacheRepositoryBase`
+* `PanacheEntity`
+
+The {ProductName} Panache JPA extension eliminates the following Hiberante ORM issues:
+
+* *Duplicate ID logic*
++
+You use the `PanacheEntity` class to create entities. Because the ID is defined automatically in the `PanacheEntity` class, you do not need to define it in your `PanacheEntity` instance. However, you can specify your own ID in your instance if you prefer.  If you require a custom ID strategy, you can extend the `PanacheEntityBase` class instead.
+
+* *Useless `getter` and `setter` functions*
++
+Because Java does not support properties, when you write Java code you must create fields and then generate `getter` and `setter` functions for those fields, even if all you need to do is read or write to the field. With Panache, if you define a field as a public field and then read or write to that field, Panache generates the required `getter` and `setter` functions. In addition, you can write other accessors as required.
+* *Entity definitions split from operations*
++
+In the traditional Java EE model, you put the the entity definition in one class, the model, and the operations performed on the model in another class, the data access object (DAO) or repository. The Panache active record method enables you to create the entity definition and the repository in the same class.
+
+* *Verbose Hibernate queries*
++
+Panache provides Hibernate Query Language (HQL) shortcuts to simplify your queries. You do not write parts of the query that you do not need.
+
+
+
+
+=== Using the Panache active record pattern
+
+
+When you use Hiberate ORM, you must create two classes that separate the state and its operations even though you would never do that for regular Java objects where state and methods are in the same class. In addition to requiring two classes, Hibernate ORM requires you to  inject the repository where you need to perform entity operations, which breaks your edit flow and requires you to get out of the code that you are  writing to set up an injection point before coming back to use it.
+
+You can use the Panache active record pattern with the `PanacheEntity` class to keep the model and the repository together.
+
+.Procedure.
+
+. Add the Panache JPA extension dependency to your {ProductName} project `pom.xml` file:
++
+[source,java]
 ----
 <dependencies>
-    <!-- Hibernate ORM specific dependencies -->
     <dependency>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-hibernate-orm-panache</artifactId>
     </dependency>
-
-    <!-- JDBC driver dependencies -->
-    <dependency>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-jdbc-postgresql</artifactId>
-    </dependency>
-</dependencies>
```

Comment:
If we show the opening tag, we should show the close tag too, no?

Reply:
Right! Fixed in two places.